src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
I think after this is in a stable release we can modify phpspec2-expects and then do a survey of other extensions to look at a controlled BC break on the Caller constructor